.nycrc

```diff
@@ -0,0 +1,6 @@
+{
+  "branches": 94,
```

Comment:
adding the config as the check coverage fails (<99%). but I think it's covered tho:

* sourceCode.xxx was covered when running eslint v8.
* context.xxx was covered when running eslint v7.